maps/cam_lao_nam/mission.sqm

```diff
@@ -106,6 +106,7 @@ addons[]=
 	"A3_Rocks_F_Blunt",
 	"A3_Rocks_F_Sharp",
 	"A3_Data_F_ParticleEffects",
+	"Blastcore_VEP",
```

Comment:
Please turn off blastcore before editing the map :) this will require everyone to have blastcore to play.